packages/meditrak-server/src/routes/exportSurveys/cellBuilders/AutocompleteConfigCellBuilder.js

```diff
@@ -12,8 +12,4 @@ export class AutocompleteConfigCellBuilder extends KeyValueCellBuilder {
     }
     return value;
   }
-
-  extractRelevantObject({ autocomplete }) {
-    return autocomplete;
-  }
```

Comment:
My apology @kael89 , I think I confused you on Friday... The `autocomplete` config was added recently into Autocomplete questions. However it is optional (so by default the config column is `{}`, but if you enable the new feature in Autocomplete questions, the config column would be `{ autocomplete: { createNew: true } }`. So that's why when you searched `config::text like '%autocomplete%'` in the current db you couldn't see any with (because data stewards haven't started to use it yet).

We can change this to the below maybe?
```
extractRelevantObject(object) {
    return object && object.autocomplete ? object.autocomplete : object;
}
```
Sorry for the hassle

Reply:
No worries @chris-bes , it makes sense now!